src/models/Person.ts

```diff
@@ -16,8 +16,8 @@ export default interface Person extends WritableDbItem {
 export enum UserRoles {
   CLIENT = 1 << 0,
   STAFF = 1 << 1,
-  DEVELOPER = 1 << 51,
-  BOT = 1 << 52,
+  DEVELOPER = 1 << 19,
+  BOT = 1 << 20,
```

Comment:
This should be correct as-is. If you're getting different values, that's a bug (looks like integer overflow to me)

Reply:
I changed following a eslint suggestion that we where out of JS safe int range witth he old values

Reply:
Nevermind reverted